gatsby-config.js

```diff
@@ -826,101 +414,13 @@ module.exports = {
                             }
                         ]
                     },
-                    {
-                        title: "Troubleshooting guides",
-                        path: "documentation/troubleshooting-guides",
-                        pages: [{
-                                title: "Overview",
-                                path: "documentation/troubleshooting-guides"
-                            },
-                            {
-                                title: "Push messaging",
-                                path: "documentation/troubleshooting-guides/push-messaging"
-                            },
-                            {
-                                title: "Push notifications",
-                                path: "documentation/troubleshooting-guides/push-notifications"
-                            }
-                        ]
-                    },
                     {
                         title: "Frequently asked questions",
                         path: "documentation/faq"
                     },
                     {
                         title: "Migrate to Swift",
                         path: "documentation/migrate-to-swift"
-                    },
-                    {
-                        title: "Privacy and GDPR",
-                        path: "documentation/privacy-and-gdpr"
-                    },
```

Comment:
we should keep the gdpr file too -> Using Adobe Experience Cloud Solution extensions section